docs/api/cjoyce-user.md

```diff
@@ -0,0 +1,48 @@
+# `user` resource
+
+Contains details about the users of the service. A user must be added to the service before they can add a [task](https://github.com/cnjoyce1225/to-do-service-au24/blob/65725a13e9c10c6daaccdf216c7dbd0d45e02c8d/docs/api/task.md).
+
+## Endpoints:
+<details><summary>Base endpoint</summary>
+
+```shell
+
+{server_url}/users
```

Comment:
```suggestion
{base_url}/users
```

This was changed in #54 and #63 to be more consistent across the docs and the Postman scripts.